consul/config.sls

```diff
@@ -0,0 +1,19 @@
+{% from "consul/map.jinja" import consul with context %}
+
+include:
+  - .install
```

Comment:
This needs to include the service state so that it can properly inject its requirements if this is the only state run. It should also probably not require the install state so that it can run faster when all that we want to do is update the config data. The install state should require the config instead.